connection.go

```diff
@@ -740,6 +740,14 @@ func (c *Connection) handleFrameNoRelay(frame *Frame) bool {
 // writeFrames is the main loop that pulls frames from the send channel and
 // writes them to the connection.
 func (c *Connection) writeFrames(_ uint32) {
+	defer func() {
+		// Remaining frames in sendCh must be drained and released,
+		// or we will leak frames
+		for len(c.sendCh) > 0 {
+			c.opts.FramePool.Release(<-c.sendCh)
```

Comment:
The code below in stopCh looks similar.

Looking at it it looks like the use-case is different (here we drop the frames, there we still send them) - but could you verify?